Source/Client/Sound/DirectSound.cs

```diff
@@ -220,52 +233,62 @@ public static void ResetPositionalSounds()
 		}
 
 		// This returns a sound object by filename
-		public static ISound GetSound(string filename, bool positional)
-		{
+		public static string GetSound(string filename, bool positional) //ISound
```

Comment:
I propose to preserve the old API: let's write our own `ISound` implementation and return it from `GetSound`:
```suggestion
		public static Sound GetSound(string filename, bool positional)
```

This way, we'll be able to leave most of the old code intact (because our `Sound` type will still have methods like `Position` and `Play`), and add support for things like positional sound later.

Reply:
Agree